src/shared/format-table.c

```diff
@@ -845,6 +848,11 @@ int table_add_many_internal(Table *t, TableDataType first_type, ...) {
                         data = &buffer.b;
                         break;
 
+                case TABLE_BOOLEAN_CHECKMARK:
+                        buffer.b = ((va_arg(ap, const char *) == special_glyph(SPECIAL_GLYPH_CROSS_MARK)) ? false : true);
+                        data = &buffer.b;
+                        break;
```

Comment:
nope, this should do the same thing as TABLE_BOOLEAN, i.e. expect an `int`. i.e. just add a `case TABLE_BOOLEAN_CHECKMARK:` as one extra line after the `case TABLE_BOOLEAN:`, that's all.

i.e. TABLE_BOOLEAN and TABLE_BOOLEAN_CHECKMARK should be read, processed, stored exacly the same as TABLE_BOOLEAN in almost all ways — except one: when *outputting* in *text* form, show the cross mark, and not the yes/no thing.